engine/crates/engine/src/registry/export_sdl.rs

```diff
@@ -1,10 +1,14 @@
 use std::fmt::Write;
 
-use super::{Deprecation, EnumType, InputObjectType, InterfaceType, ObjectType, ScalarType, UnionType};
+use super::{
+    field_set::FieldSetDisplay, Deprecation, EnumType, InputObjectType, InterfaceType, ObjectType, RegistrySdlExt,
+    ScalarType, UnionType,
+};
 use crate::registry::{MetaField, MetaInputValue, MetaType, Registry};
 
-impl Registry {
-    pub fn export_sdl(&self, federation: bool) -> String {
+// TODO: Delete this when we can
+impl RegistrySdlExt for Registry {
```

Comment:
why change it at all since it's for the old registry?

Reply:
Well, it's still needed I _think_ - there's certainly places that still use the Registry, and I think there are _at least_ some tests that need SDL printing for the old registry.

I made it an extension trait because the registries no longer live in the `engine`, so it can't just be a normal impl anymore.  (Arguably I could have moved it to registry-v1 or registry-v2, but 🤷).  And afaik the rest of the changes in this file are just a result of not being able to have things in an impl.